templates/teku.yml.j2

```diff
@@ -42,7 +42,7 @@ eth1-endpoint: "{{teku_deposit_eth1_endpoint}}"
 #Xtransaction-record-directory: "/tmp/teku"
 
 # logging
-{% if teku_log4j_config_file != "" %}
+{% if teku_log4j_config_file == "" %}
 #log-colour-enabled: False
```

Comment:
We probably want log-colour-enabled: False regardless.  It seems to default to on even with custom log config.

Reply:
:+1: fixed